great_expectations/render/renderer/inline_renderer.py

```diff
@@ -215,30 +215,3 @@ def get_rendered_content(
         )
 
         return self._get_atomic_rendered_content_for_object(render_object=render_object)
-
-    @staticmethod
-    def replace_or_keep_existing_rendered_content(
```

Comment:
Talked to @NathanFarmer and we agreed this should be safe to remove. This function was largely just to abstract away replacing missing rendered_content with new rendered content, plus a bit of extra logic about the failed renderer_types that seem to have been preempting a situation that isn't driven by needs of the business logic.